snapshot.go

```diff
@@ -71,10 +71,10 @@ func (sn Snapshot) ID() backend.ID {
 func (sn *Snapshot) fillUserInfo() error {
 	usr, err := user.Current()
 	if err != nil {
-		return err
+		return nil
```

Comment:
If the current user cannot be determined, do not fill in any user info. This information isn't  required for restic to function (it's just an information after all, nothing that you'd really need to restore a backup), so don't fail hard when `user.Current()` returns an error.